cmd/tealdbg/cdtSession.go

```diff
@@ -66,7 +66,7 @@ func makeCdtSession(uuid string, debugger Control, ch chan Notification) *cdtSes
 }
 
 func (s *cdtSession) sourceMapHandler(w http.ResponseWriter, r *http.Request) {
-	sm, err := s.debugger.GetSourceMap()
+	sm, err := GetSourceMap(s.debugger)
```

Comment:
I supposed it could be but I made the original method take the interface instead of having the session receiver. I _thought_ that was what you had suggested  but may have misunderstood

Reply:
I do not remember details, sorry. I probably thought about having a common code generating source maps from different underlying data. But here `session` object has all the data and own code for maps creation.